specification/automanage/resource-manager/readme.python.md

```diff
@@ -15,11 +15,21 @@ python:
   package-version: 0.1.0
   clear-output-folder: true
 ```
+
+```yaml $(python) && $(track2) 
+azure-arm: true 
+license-header: MICROSOFT_MIT_NO_VERSION 
+package-name: azure-mgmt-automanage
+package-version: 0.1.0-preview
+clear-output-folder: true 
+no-namespace-folders: true 
+``` 
+
 ``` yaml $(python) && $(python-mode) == 'update'
-python:
```

Comment:
Hi @abhinav-ghai  ,please keep this block there.
Refer to this for python.md configuration: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/appconfiguration/resource-manager/readme.python.md

Reply:
Thanks, fixed.